src/Character/CharacterFactory.java

```diff
@@ -1,76 +1,220 @@
-		case Lookout: return new Lookout(player);
-		//case Marshall: return new Marshall(player);
-		//case Mason: return new Mason(player);
-		//case MasonLeader: return new MasonLeader(player);
+		case Jailor:
+			break;
+		case Janitor:
+			break;
+		case Jester:
+			break;
+		case Judge:
+			break;
+		case Kidnapper:
+			break;
+		case Liaison:
+			break;
+		case Lookout:
+			break;
+		case Mafioso:
+			break;
+		case Marshall:
+			break;
+		case Mason:
+			break;
+		case MasonLeader:
+			break;
+		case MassMurderer:
+			break;
 		case Mayor: return new Mayor(player);
-		case Sheriff: return new Sheriff(player);
-		//case Spy: return new Spy(player);
-		//case Stump: return new Stump(player);
-		case Veteran: return new Veteran(player);
+		case SerialKiller:
+			break;
+		case Sheriff:
+			break;
+		case Silencer:
+			break;
+		case Spy:
+			break;
+		case Stump:
+			break;
+		case Survivor:
+			break;
+		case Vanguard:
+			break;
+		case Veteran:
+			break;
 		case Vigilante: return new Vigilante(player);
-		//case Agent: return new Agent(player);
-		//case Beguiler: return new Beguiler(player);
-		//case Blackmailer: return new Blackmailer(player);
-		case Consigliere: return new Consigliere(player);
-		case Consort: return new Consort(player);
-		//case Disguiser: return new Disguiser(player);
-		//case Framer: return new Framer(player);
-		case Godfather: return new Godfather(player);
-		//case Janitor: return new Janitor(player);
-		//case Kidnapper: return new Kidnapper(player);
-		case Mafioso: return new Mafioso(player);
-		//case Administrator: return new Administrator(player);
-		//case Deceiver: return new Deceiver(player);
-		//case DragonHead: return new DragonHead(player);
-		//case Enforcer: return new Enforcer(player);
-		//case Forger: return new Forger(player);
-		//case IncenseMaster: return new IncenseMaster(player);
-		//case Informant: return new Informant(player);
-		//case Interrogator: return new Interrogator(player);
-		//case Liaison: return new Liaison(player);
-		//case Silencer: return new Silencer(player);
-		//case Vanguard: return new Vanguard(player);
-		//case Amnesiac: return new Amnesiac(player);
-		case Arsonist: return new Arsonist(player);
-		//case Auditor: return new Auditor(player);
-		case Cultist: return new Cultist(player);
-		//case Executioner: return new Executioner(player);
-		//case Jester: return new Jester(player);
-		//case Judge: return new Judge(player);
-		case MassMurderer: return new MassMurderer(player);
-		case SerialKiller: return new SerialKiller(player);
-		//case Survivor: return new Survivor(player);
-		case Witch: return new Witch(player);
-		//case WitchDoctor: return new WitchDoctor(player);
+		case Witch:
+			break;
+		case WitchDoctor:
+			break;
 		default:
 			break;
+		
 		}
 		return null;
 	}
 	
+	/**
+	 * Enumeration of all the roles in the game.
+	 */
+	public enum Roles {
+
+		Bodyguard(Sides.Town, "Bodyguard"), BusDriver(Sides.Town, "BusDriver"), Citizen(
```

Comment:
I have a good yet bad habit to ctrl shift f (auto format) every two seconds, and that's what comes out. I'm sorry. Will change.